src/hotspot/share/utilities/growableArray.hpp

```diff
@@ -141,6 +141,14 @@ class GrowableArrayView : public GrowableArrayBase {
     return !(*this == rhs);
   }
 
+  E* data() {
+    return _data;
+  }
+
+  const E* data() const {
+    return _data;
+  }
+
```

Comment:
This accessor is added to be able to temporarily view a stable GrowableArray instance as a C-style array. It is used to by `NativeCallConv` and `RegSpiller` in `foreign_globals.hpp`. GroableArray already has an `adr_at` accessor that does basically the same thing, but using `adr_at(0)` fails on empty growable arrays since it also performs a bounds check.

Reply:
I'll try migrating to `GrowableArray`